packages/cosmic-swingset/lib/chain-main.js

```diff
@@ -103,7 +105,7 @@ export default async function main(progname, args, { path, env, agcc }) {
 
   // We try to find the actual cosmos state directory (default=~/.ag-chain-cosmos), which
   // is better than scribbling into the current directory.
-  const cosmosHome = getFlagValue('home', `${env.HOME}/.ag-chain-cosmos`);
+  const cosmosHome = getFlagValue('home', `${os.homedir()}/.ag-chain-cosmos`);
```

Comment:
`env` was passed explicitly; can we do the same for `homedir`, please? I think this file _does_ use ocap discipline.

Reply:
Thanks for pointing that out; it helps.

It will probably take me another month or so to start becoming allergic to non-ocap discipline.

Reply:
How can we induce that allergy to happen faster ;)